rj_gameplay/rj_gameplay/tactic/temp_pivot_tactic.py

```diff
@@ -0,0 +1,96 @@
+"""Contains the stub for the move tactic. """
+
+from dataclasses import dataclass
+from typing import List, Optional
+from typing import Dict, Generic, List, Optional, Tuple, Type, TypeVar
+
+import stp.action as action
+import stp.rc as rc
+import stp.tactic as tactic
+import stp.role as role
+
+import rj_gameplay.eval
+import rj_gameplay.skill as skills
+from rj_gameplay.skill import temp_pivot_skill
+import stp.skill as skill
+import numpy as np
+
+
+class pivot_cost(role.CostFn):
+    """
+    A cost function for how to choose a striker
+    TODO: Implement a better cost function
+    """
+    def __init__(self, target_point : np.ndarray):
+        self.target_point = target_point
+
+    def __call__(
+        self,
+        robot: rc.Robot,
+        prev_result: Optional["RoleResult"],
+        world_state: rc.WorldState,
+    ) -> float:
+    
+        if robot.id == 7:
+            return 0.0
+        return 1.0
+
+class Pivot(tactic.ITactic):
+    """
+    A striker tactic which captures then shoots the ball
+    """
+
+
+    def __init__(self, target_point : np.ndarray):
+        self.pivot = tactic.SkillEntry(temp_pivot_skill.Pivot(pivot_point = [0.0,0.0],target_point = target_point))
+        self.cost = pivot_cost(target_point)
+        
+    def compute_props(self):
+        pass
+
+    def create_request(self, **kwargs) -> role.RoleRequest:
+        """Creates a sane default RoleRequest.
+        :return: A list of size 1 of a sane default RoleRequest.
+        """
+        pass
+
+    def get_requests(
+        self, world_state: rc.WorldState, props
+    ) -> List[tactic.RoleRequests]:
+        """ Checks if we have the ball and returns the proper request
+        :return: A list of size 1 of role requests
+        """
+
+        role_requests: tactic.RoleRequests = {}
+
+        pivot_request = role.RoleRequest(role.Priority.HIGH, True, self.cost)
+        # has_ball = True
+        # for robot in world_state.our_robots:
+        #     if robot.has_ball_sense:
+        #         has_ball = True
+        # if has_ball:
+        #     role_requests[self.shoot] = [striker_request]
+        #     role_requests[self.capture] = []
+        # else:
+        #     role_requests[self.capture] = [striker_request]
+        #     role_requests[self.shoot] = []
+        # role_requests
```

Comment:
Please don't add dead code.

Reply:
To be fair, I think this whole file will get deleted anyways, since it's a test wrapper for the Pivot skill.